docs/snippets/MULTI0.cpp

```diff
@@ -1,18 +1,18 @@
-#include <ie_core.hpp>
+#include <openvino/openvino.hpp>
 
 int main() {
 using namespace InferenceEngine;
 //! [part0]
-    Core ie; 
-    auto network = ie.ReadNetwork("sample.xml");
-    //NEW IE-CENTRIC API, the "MULTI" plugin is (globally) pre-configured with the explicit option:
-    ie.SetConfig({{"MULTI_DEVICE_PRIORITIES", "HDDL,GPU"}}, "MULTI");
-    ExecutableNetwork exec0 = ie.LoadNetwork(network, "MULTI", {});
+ov::Core core;
+std::shared_ptr<ov::Model> model = core.read_model("sample.xml");
+//NEW IE-CENTRIC API, the "MULTI" plugin is (globally) pre-configured with the explicit option:
```

Comment:
remove NEW IE-CENTRIC API everywhere. It's obsolete comments.

Reply:
fixed